tutorials/noise_configuration.md

```diff
@@ -0,0 +1,49 @@
+# Tutorials to Configure Noisepy Job
```

Comment:
Why not have an actual `config.yml` file instead so people don't have to copy/paste from it? 

Having it as a file also allows us to use it when testing in the CI pipeline.

Reply:
there i just wanted to explain what the variables are, i think even the config.yml file can have the comments so that users know how to change them. Some parameters should be explained in this MD file such as frequency and time normalization